OutOfSchool/OutOfSchool.WebApi/Services/ChildService.cs

```diff
@@ -219,22 +219,29 @@ public async Task<SearchResult<ChildDto>> GetByParentIdOrderedByFirstName(Guid p
     }
 
     /// <inheritdoc/>
-    public async Task<SearchResult<ChildDto>> GetByUserId(string userId, OffsetFilter offsetFilter)
+    public async Task<SearchResult<ChildDto>> GetByUserId(string userId, bool isGetParent, OffsetFilter offsetFilter)
     {
         this.ValidateUserId(userId);
         this.ValidateOffsetFilter(offsetFilter);
 
         logger.LogDebug($"Getting Child's for User started. Looking UserId = {userId}.");
 
-        var totalAmount = await childRepository.Count(x => x.Parent.UserId == userId).ConfigureAwait(false);
+        Expression<Func<Child, bool>> predicate = x => x.Parent.UserId == userId;
+
+        if (!isGetParent)
+        {
+            predicate = predicate.And(x => x.IsParent == isGetParent);
+        }
```

Comment:
We can write it shorter ))))
predicate = predicate.And(x => !x.IsParent);

Reply:
I agree. I'm doing it.